helpers/converter.js

```diff
@@ -0,0 +1,21 @@
+var helpers = require("./core");
```

Comment:
This technically isn't a helper the same way other things are helpers.  Might have just put in the `src` folder.